src/main/java/io/cdap/e2e/pages/actions/CdfPluginPropertiesActions.java

```diff
@@ -141,23 +141,33 @@ public static void changeDataTypesInOutputSchema(String jsonFieldList) {
    */
   public static void clickGetSchemaButton() {
     ElementHelper.clickOnElement(CdfPluginPropertiesLocators.getSchemaButton);
+    WaitHelper.waitForElementToBeDisabled(CdfPluginPropertiesLocators.getSchemaButton);
+    /*
+    TODO : Remove the commented lines of code once the bug is resolved by Chrome Team.
+    Bug : https://cdap.atlassian.net/browse/PLUGIN-1693
     WaitHelper.waitForElementToBeOptionallyDisplayed(
       CdfPluginPropertiesLocators.locatorOfLoadingSpinnerOnGetSchemaButton(), ConstantsUtil.SMALL_TIMEOUT_SECONDS);
     WaitHelper.waitForElementToBeHidden(
       CdfPluginPropertiesLocators.locatorOfLoadingSpinnerOnGetSchemaButton(), ConstantsUtil.DEFAULT_TIMEOUT_SECONDS);
     WaitHelper.waitForElementToBeDisplayed(CdfPluginPropertiesLocators.getSchemaButton);
+    */
   }
 
   /**
    * Click on the Validate button inside Plugin's properties page
    */
   public static void clickValidateButton() {
     ElementHelper.clickOnElement(CdfPluginPropertiesLocators.validateButton);
+    WaitHelper.waitForElementToBeDisabled(CdfPluginPropertiesLocators.validateButton);
```

Comment:
We should follow this up with `WaitHelper.waitForElementToBeEnabled`
essentially checking click->disabled(gif)->enabled

Reply:
yes now we have changed it to WaitHelper.waitForElementToBeOptionallyDisabled . Here we are waiting till the element is disabled before displayed and clickable again .